docs/quickstart-guide.md

```diff
@@ -78,16 +78,16 @@ $ kubectl get gatling,job,pod
 Expected output would be like this:
 
 ```
-NAME                                                      AGE
-gatling.gatling-operator.tech.zozo.com/gatling-sample01   10s
+NAME                                                      RUNNED   REPORTED   NOTIFIED   REPORTURL                                                                                 AGE
+gatling.gatling-operator.tech.zozo.com/gatling-sample01   0/3                            https://gatling-operator-reports.s3.amazonaws.com/gatling-sample01/318807881/index.html   24s
 
 NAME                                COMPLETIONS   DURATION   AGE
-job.batch/gatling-sample01-runner   0/3           9s         9s
+job.batch/gatling-sample01-runner   0/3           24s        24s
 
 NAME                                READY   STATUS    RESTARTS   AGE
-pod/gatling-sample01-runner-8rhl4   1/1     Running   0          9s
-pod/gatling-sample01-runner-cg8rt   1/1     Running   0          9s
-pod/gatling-sample01-runner-tkplh   1/1     Running   0          9s
+pod/gatling-sample01-runner-8rhl4   2/2     Running   0          24s
+pod/gatling-sample01-runner-cg8rt   2/2     Running   0          24s
+pod/gatling-sample01-runner-tkplh   2/2     Running   0          24s
```

Comment:
Gatling runner Pod is no longer a single container but multi-containers from this update https://github.com/st-tech/gatling-operator/pull/65